tests/acceptance/async_adapter_tests.py

```diff
@@ -84,22 +87,22 @@ class TestConsumeCancel(AsyncTestCase, AsyncAdapters):
 
     def begin(self, channel):
         self.queue_name = self.__class__.__name__ + ':' + uuid.uuid1().hex
-        channel.queue_declare(self.on_queue_declared, queue=self.queue_name)
+        channel.queue_declare(callback=self.on_queue_declared, queue=self.queue_name)
```

Comment:
Move `callback=self.on_queue_declared` to the end to match the new arg order convention (even if kwarg works around the ordering issue) ? Elsewhere in this file too. Test code snippets get copied to apps and other tests.